AFQ/segmentation.py

```diff
@@ -3,6 +3,7 @@
 from scipy.spatial.distance import mahalanobis, cdist
 
 import nibabel as nib
+from tqdm import tqdm
```

Comment:
If you use
```
from tqdm.auto import tqdm
```
then tqdm will automatically detect the backend and choose between console and notebook versions so that the progress bar is pretty in all cases.

Reply:
Ooh! Good to know!